src/components/NavMenu/NavMenu.jsx

```diff
@@ -1,91 +1,107 @@
 /* eslint-disable react/require-default-props */
 import React, { useState, createElement } from 'react';
-import { Link, useLocation } from 'react-router-dom';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
 import { Menu, Layout, Button } from 'antd';
-import PropTypes from 'prop-types';
+import { instanceOf } from 'prop-types';
 import {
   ProfileOutlined,
```

Comment:
not sure if the `ProfileOutlined` icon is still being used in this file (says it got removed compared to the original) but in general if we're not using something we're importing we might as well just not import it at all

Reply:
used for /hours (but idek if we're still using hours log at all)